doc/source/serve/advanced-guides/dev-workflow.md

```diff
@@ -79,6 +79,18 @@ After you're done testing, you can shut down Ray Serve by interrupting the `serv
 
 Note that rerunning `serve run` will redeploy all deployments. To prevent redeploying those deployments whose code hasn't changed, you can use `serve deploy`; see the [Production Guide](serve-in-production) for details.
```

Comment:
```suggestion
Note that rerunning `serve run` redeploys all deployments. To prevent redeploying the deployments whose code hasn't changed, you can use `serve deploy`; see the [Production Guide](serve-in-production) for details.
```